Assets/Characters/Enemy/SquareEnemy.cs

```diff
@@ -0,0 +1,18 @@
+using Core.CharacterTypes;
+using Core.Enums;
+using UnityEngine;
+
+public class SquareEnemy : Enemy
+{
+    private void Awake()
+    {
+        Id = CharacterId.Enemy1;
+        MinExp = 2;
+        MaxExp = 10;
+    }
```

Comment:
Maybe export these values to some file which holds constants.
It would also be nice if we could change them whenever we want from the inspector without having to change the code, we should figure out how to do this (not urgent for this PR of course)